celery/app/routes.py

```diff
@@ -24,10 +24,10 @@
 
 GLOB_PATTERNS = ('*', '?', '[', ']', '!')
 
-MAP_ROUTES_MUST_BE_A_DICTIONARY = """\
-Starting from Celery 5.1 the task_routes configuration must be a dictionary.
-Support for providing a list of router objects will be removed in 6.0.
-""".strip()
+MAP_ROUTES_MUST_BE_A_DICTIONARY = (
+    "Starting from Celery 5.1 the task_routes configuration must be a dictionary. "
+    "Support for providing a list of router objects will be removed in 6.0."
```

Comment:
@xirdneh there's no newline here.
Also, why is this better?

Reply:
Good point about the new line, it should be added there.

This change was marked as a *nit* because either way works and I'm OK with both versions.
But, I find using `strip()` at the end of a multi line string unnecessary.